static/css/layout.css

```diff
@@ -15,5 +23,78 @@
 }
 
 .input_form {
-    padding-left: 10%;
-}
\ No newline at end of file
+    display: block;
+    padding: 30px 0 50px 0;
+    text-align: center;
+    margin-bottom: 35px;
+}
+
+form {
+    background-color: #ededff;
+    border: 2px solid #dcdcff;
+    box-shadow: 0 0 5px rgba(1, 1, 1, 0.2);
+    border-radius: 8px;
+    padding: 15px;
+    display: inline-block;
+    margin-left: auto;
+    margin-right: auto;
+    text-align: left;
+}
+
+label {
+    display: block;
+    width: 100px;
+    text-align: left;
+    padding-right: 10px;
+    font-size: 20px;
+    width: 100%;
+}
+
+input[type=text] {
+    border: 2px solid black;
+    border-radius: 8px;
+    padding: 5px;
+    margin-bottom: 5px;
+    width: 100%;
+}
+
+input[type=date] {
+    border: 2px solid black;
+    border-radius: 9px;
+    padding: 5px;
+    margin-bottom: 5px;
+    width: 100%;
+}
+
+input:hover {
+    cursor: pointer;
+}
```

Comment:
I would prefer to leave the cursor as a text cursor when hovering over the chart name field, but otherwise I like this behavior. Can you move this so that it only applies to the date input?